Source/SwiftyDropbox/Platform/SwiftyDropbox_macOS/OAuthDesktop.swift

```diff
@@ -86,11 +86,12 @@ public class DesktopSharedApplication: SharedApplication {
     public static var sharedDesktopApplication: DesktopSharedApplication?
 
     let sharedWorkspace: NSWorkspace
-    let controller: NSViewController?
+    weak var controller: NSViewController?
```

Comment:
the weak controller _might_ cause problems... I remember seeing it and wanting to change it, can't remember why I left it alone

Reply:
This is in their native Mac code which we don't use since we use Catalyst now. The iOS platform code they have is all we need to worry about.

Reply:
OAuthMobile has an identical change, wasn't paying attention to which one I commented on, sorry

Reply:
Oh I see. Yea I think the weak controller could mean that the authorization flow would fail to present something if we dismissed the view controller we tried to present the login on, which I thought might be possible on Mac, but I haven't found anything broken with my testing. In the PR for Notability I'll make sure QA knows to look for regressions of this nature.

Reply:
PR for reference: https://github.com/Ginger-Labs/Notability/pull/4219